rust/stackablectl/src/cli/mod.rs

```diff
@@ -92,25 +87,30 @@ impl Cli {
     /// Returns a list of demo files, consisting of entries which are either a path or URL. The list of files combines
     /// the default demo file URL, [`REMOTE_DEMO_FILE`], files provided by the ENV variable [`ENV_KEY_DEMO_FILES`], and
     /// lastly, files provided by the CLI argument `--demo-file`.
-    pub fn get_demo_files(&self) -> Result<Vec<PathOrUrl>, PathOrUrlParseError> {
-        let mut files = get_files(REMOTE_DEMO_FILE, ENV_KEY_DEMO_FILES)?;
+    pub fn get_demo_files(&self, branch: &str) -> Result<Vec<PathOrUrl>, PathOrUrlParseError> {
```

Comment:
note: The function should now mention the new `branch` parameter it accepts and what it does.

Reply:
Done in e6f73ca.